### PR TITLE
chore(documentation): documentation and manuals for ZITADEL

### PR DESCRIPTION
**Pull request by fforootd**
This PR merges the WIP of the documentation, so that it can be used in the master branch to further document the features

### CODE REVIEW

File: site/docs/administrate/03-projects.en.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+title: Projects
+---
+
+### What are projects
+
+The idea of projects is to have a vessel for all components who are closely related to each other.
+In ZITADEL all clients located in the same project share their roles, grants and authorizations.
+From a access management perspective you manage who has what role in the project and your application consume this information.
+A project belongs to exactly one organisation.
+
```
Comment (fgerschwiler) [inline site/docs/administrate/03-projects.en.md:11]:
```suggestion
The attribute project role assertion defines, if the roles should be integrated in the tokens without sending corresponding scope (urn:zitadel:iam:org:project:role:{rolename})
With the project role check you can define if a user should have a requested role to be able to logon.

```

**Review by fgerschwiler — CHANGES_REQUESTED** [review]

File: site/docs/administrate/01-console.en.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+### What is Console
+
+Console is the ZITADEL Graphical User Interface.
+
+**Users** can manage some information by their own.
+
+- profile information
+- credentials
+- identity providers
```
[the remark this answers is not in the document]
Comment (fforootd) [reply site/docs/administrate/01-console.en.md:13]:
```suggestion
- external logins
```

File: site/docs/administrate/02-organisations.en.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+#### Global organisation
+
+ZITADEL provides a global organisation for users who manage their accounts on their own. Think of this like the difference between a "Microsoft Live Login" vs. "AzureAD User"
+or if you think of Google "Gmail" vs "Gsuite".
+
+### Create an organisation without existing login
+
+ZITADEL allows you to create a new organisation without a pre-existing user. For [ZITADEL.ch](https://zitadel.ch) you can create a org by visiting the [Register organisation](https://accounts.zitadel.ch/register/org)
+
+> Screenshot here
```
[the remark this answers is not in the document]
Comment (fforootd) [reply site/docs/administrate/02-organisations.en.md:19]: I will integrate some basic screenshots

File: site/docs/administrate/03-projects.en.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+**Clients**
+
+Clients are described here [What are clients](administrate#What_are_clients)
+Basically these are you applications who initiate the authorization flow
+
+**Roles**
+
+Roles (or Project Roles) is a mean of managing users access rights for a certain project.
+These roles are opaque for ZITADEL and have no weight in relation to each other.
+So if a user has two roles, admin and user in a certain project, the information will be treated additive.
```
Comment (fforootd) [inline site/docs/administrate/03-projects.en.md:21]:
```suggestion
**Roles**

[Roles (or Project Roles)](administrate#Roles) is a mean of managing users access rights for a certain project.
These roles are opaque for ZITADEL and have no weight in relation to each other.
So if a user has two roles, admin and user in a certain project, the information will be treated additive.
```

File: site/docs/administrate/05-roles.en.md [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+
+- [OpenID Connect / OAuth](integrate#How_to_consume_authorizations_in_your_application_or_service)
+
+### Manage Roles
+
+Each **role** consist of three fields.
+
+| Field        | Description                                                                  | Example                                          |
+|:-------------|:-----------------------------------------------------------------------------|--------------------------------------------------|
+| Key          | This is the **Roles** actual name                                            | User                                             |
```
Comment (fforootd) [inline site/docs/administrate/05-roles.en.md:21]:
```suggestion
| Key          | This is the **Roles** actual name which can be used to verify the users roles.                                            | User                                             |
```

**Review by fforootd — COMMENTED** [review]
Integrated comments.

Screenshots will be made after the merge

**Review by fgerschwiler — APPROVED** [review]

**Pull request merged**